partner_attribute_hls/models/res_partner.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2021 Quartile Limited
```

Comment:
```suggestion
# Copyright 2021 Quartile Limited
# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
```
Sorry, here as well.  You may feel that it's redundant, however it's just how it should be done according to the convention - better to be explicit than otherwise.